containers/product-edit/preview.vue

```diff
@@ -18,18 +50,53 @@
       </div>
     </Card>
     <div class="actions">
-      <Button danger label="clear fields" />
+      <Button @click="$store.commit('newProduct/reset')" danger label="clear fields" />
```

Comment:
Better use mapActions and call the function reset instead of accessing $store

Reply:
I would've done that if there was more than one action (it's called mapAction**s** and not Action 😜). Not really worth the complication for just one method, imho